src/main/java/com/theironyard/invitator/Event.java

```diff
@@ -0,0 +1,158 @@
+package com.theironyard.invitator;
+
+import java.sql.*;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by melmo on 12/22/16.
+ */
+public class Event {
+    private Integer eventId;
+    private Integer userId;
+    private String name;
+    private String date;
+    private String time;
+    private String location;
+    private String description;
+    private Map<Integer, Person> invitedList = new HashMap<>();
+    private Map<Integer, Person> notInvitedList = new HashMap<>();
+
+    public Event(Integer eventId, Integer userId, String name, String date, String time, String location, String description) {
+        this.eventId = eventId;
+        this.userId = userId;
+        this.name = name;
+        this.date = date;
+        this.time = time;
+        this.location = location;
+        this.description = description;
+    }
+
+    public Event(String name, String date, String time, String location, String description, Integer userId){
+        this.eventId = 0;
+        this.userId = userId;
+        this.name = name;
+        this.date = date;
+        this.time = time;
+        this.location = location;
+        this.description = description;
+    }
+
+    public void saveEvent() throws SQLException {
+        Connection connection = DriverManager.getConnection("jdbc:postgresql:invitation");
+        PreparedStatement insert = connection.prepareStatement(
+                "INSERT INTO events (name, date, time, location, description, user_id) VALUES (?,?,?,?,?,?)",
+                Statement.RETURN_GENERATED_KEYS);
+        insert.setString(1, this.name);
+        insert.setString(2, this.date);
+        insert.setString(3, this.time);
+        insert.setString(4, this.location);
+        insert.setString(5, this.description);
+        insert.setInt(6, this.userId);
+        insert.executeUpdate();
+        ResultSet rs = insert.getGeneratedKeys();
+        rs.next();
+        this.eventId = rs.getInt("event_id");
```

Comment:
If you are creating a connection you should make sure it closes when you are done using it.